### PR TITLE
Microsoft.VC++2010Redist-x64 version 10.0.40219.325

### PR DESCRIPTION
**Pull request by tiziodcaio**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/14153)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
```
-----------------------------------------------------
Manifest: manifests\m\Microsoft\VC++2010Redist-x64\10.0.40219.325
-----------------------------------------------------
URI: https://www.microsoft.com/en-us/download/details.aspx?id=13523, Validation result: Failed, Http status code: ServiceUnavailable 
URI: https://download.microsoft.com/download/1/6/5/165255E7-1014-4D0A-B094-B6A430A6BFFC/vcredist_x64.exe, Validation result: Passed

****************************************************************************************
Manifest Content Validation
****************************************************************************************
Status: Completed
##[error] Result: Failed
```

File: manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.VC++2010Redist-x64
+PackageVersion: 10.0.40219.325
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft Visual C++ 2010 Service Pack 1 Redistributable Package MFC Security Update (x64)
+PackageUrl: https://www.microsoft.com/en-us/download/details.aspx?id=13523
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml:8]:
```suggestion
PackageName: Microsoft Visual C++ 2010 x64 Redistributable
PackageUrl: https://www.microsoft.com/en-us/download/details.aspx?id=26999
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
I see you closed the PR, but you can fix it with this

File: manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml (hunk 1 above)
Comment (tiziodcaio) [reply manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml:8]: it doesn't work!

Comment (tiziodcaio) [reply manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml:8]: wait

**Comment by OfficialEsco** [comment]
It didn't work cause you didn't add my changes

**Comment by tiziodcaio** [comment]
Can we rerun the bots now I added the changes?

**Comment by OfficialEsco** [comment]
```
-----------------------------------------------------
Manifest: manifests\m\Microsoft\VC++2010Redist-x64\10.0.40219.325
-----------------------------------------------------
URI: https://www.microsoft.com/en-us/download/details.aspx?id=13523, Validation result: Failed, Http status code: ServiceUnavailable 
URI: https://download.microsoft.com/download/1/6/5/165255E7-1014-4D0A-B094-B6A430A6BFFC/vcredist_x64.exe, Validation result: Passed

****************************************************************************************
Manifest Content Validation
****************************************************************************************
Status: Completed
##[error] Result: Failed
```

> Can we rerun the bots now I added the changes?

Just be sure that the changes are added and then push another commit

File: manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml:6]: There should be a button called add changes or something along those lines, you've just clicked "Resolved" without resolving the problem

**Comment by tiziodcaio** [comment]
> 
> 
> ```
> -----------------------------------------------------
> Manifest: manifests\m\Microsoft\VC++2010Redist-x64\10.0.40219.325
> -----------------------------------------------------
> URI: https://www.microsoft.com/en-us/download/details.aspx?id=13523, Validation result: Failed, Http status code: ServiceUnavailable 
> URI: https://download.microsoft.com/download/1/6/5/165255E7-1014-4D0A-B094-B6A430A6BFFC/vcredist_x64.exe, Validation result: Passed
> 
> ****************************************************************************************
> Manifest Content Validation
> ****************************************************************************************
> Status: Completed
> ##[error] Result: Failed
> ```
> 
> > Can we rerun the bots now I added the changes?
> 
> Just be sure that the changes are added and then push another commit

I've done a commit

**Pull request merged**